### PR TITLE
Fix Broken Texts

### LINKED ISSUE #891: Texts broken? 48 + 40
check 48 + 40

something like the following will not work, the message is added twice plus what happens to the pragma?

@ibnbr 

```abap
    ls_message-test = myname.
    ls_message-code = '001'.
    ls_message-kind = c_error.
    ls_message-pcom = '"#EC CI_DEFAULT_KEY'.
    INSERT ls_message INTO TABLE scimessages.

    enable_rfc( ).

    insert_scimessage(
        iv_code = '001'
        iv_text = 'DEFAULT KEY, add table key or EMPTY KEY'(m01) ).
```

### PR DESCRIPTION
**Pull request by ibnbr**
fix #891

### CODE REVIEW

File: src/checks/zcl_aoc_check_69.clas.abap [hunk 1]
```diff
@@ -901,12 +901,10 @@ CLASS ZCL_AOC_CHECK_69 IMPLEMENTATION.
```
Comment (larshp) [inline src/checks/zcl_aoc_check_69.clas.abap]: and the part above here can be deleted

File: src/checks/zcl_aoc_check_48.clas.abap [hunk 2]
```diff
@@ -135,7 +135,8 @@ CLASS ZCL_AOC_CHECK_48 IMPLEMENTATION.
```
Comment (larshp) [inline src/checks/zcl_aoc_check_48.clas.abap]: and the part above here can be deleted

Comment (ibnbr) [reply src/checks/zcl_aoc_check_48.clas.abap]: Done.

**Comment by larshp** [comment]
thanks 👍

**Pull request merged**